plugins/coral-plugin-offtopic/client/containers/OffTopicStatus.js

```diff
@@ -0,0 +1,6 @@
+import {PLUGIN_NAME, DEFAULT_CONFIG} from '../constants';
+
+export default function isEnabled(props = {}) {
```

Comment:
I added a comment here https://github.com/coralproject/talk/pull/726 figuring out a simpler solution to do what you need, without making the core api more complex.